openmetadata-ui/src/main/resources/ui/src/components/ActivityFeed/ActivityFeedCard/FeedCardBody/FeedCardBodyV1.tsx

```diff
@@ -184,20 +195,35 @@ const FeedCardBodyV1 = ({
               </Col>
             </Row>
             <Row>
-              <Col span={24}>
+              <Col
+                className={`${feed.type === ThreadType.Announcement && 'mt-4'}`}
+                span={24}>
                 <Typography.Text className="font-semibold">
                   {message}
                 </Typography.Text>
               </Col>
             </Row>
-            <Row>
-              <Col span={24}>
-                <RichTextEditorPreviewer
-                  className="text-wrap"
-                  markdown={announcement.description ?? ''}
-                />
-              </Col>
-            </Row>
+            {feed.type === ThreadType.Announcement && (
+              <Row>
+                <Col
+                  className={`${
+                    feed.type === ThreadType.Announcement && 'mb-1'
+                  }`}
```

Comment:
I don't think we need this condition here, as we are already in the Announcement condition ?